src/Controller/DashboardAction.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Controller;
```

Comment:
Not a blocker, but I would prefer the `..\Action` namespace for actions.

Reply:
Then maybe also change the name of service file to actions?